src/Containers/VendorSelectCard/VendorSelectCard.scss

```diff
@@ -37,18 +41,33 @@
   padding: 5px;
   font-size: 1em;
   border: 2px solid #3B3D39;
-  background-color: #F0F0EF;
+  letter-spacing: 0.5px;
+  // background-color: #F0F0EF;
+  background-color: #E0CDAF;
   outline: 0;
   border-radius: 10px;
   cursor: pointer;
 }
 
+#edit{
+  background-color: #E0CDAF;
```

Comment:
I think that it looks a little off to have the background color on hover be the same as the background color of the whole thing on hover - that almost makes it seem like the button is disabled to me